/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Main.java to edit this template
 */
package parttwo;

import java.util.Scanner;

/**
 *
 * @author Student
 */
public class PartTwo {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        // TODO code application logic here
        Message msg = new Message();
        Scanner scan = new Scanner(System.in);
        
        int option;
        
        do{
            System.out.println("n===== Select any of the option below =====");
            System.out.println("1. Send message");
            System.out.println("2. Coming soon");
            System.out.println("3. Exit");
            
            option = scan.nextInt();
            
            switch(option){
                case 1 -> msg.sendingMessage();
                case 2 -> System.out.println();
                case 3 -> System.out.println();
                default -> System.out.println("Invalid option!!");
            
            }
        }while(option != 3);
    }
}
